templates/root.lucius

```diff
@@ -0,0 +1,141 @@
+@linkcolor: #E74C3D;
+@linkbordercolor: #F7C4BF;
+@lightgray: #D7D7D7;
+@mediumgray: #615E5F;
+@darkgray: #221E1F;
+
+body {
+  font-family: "Lato", "Gill Sans", "Gill Sans MT", sans-serif;
+}
+
+a {
+  color: #{linkcolor};
+  text-decoration: none;
+}
+
+a:hover {
+  border-bottom: 2px solid #{linkbordercolor};
+}
+
+h1, h2, h3, h4, h5, h6 {
+  color: #{darkgray};
+}
+
+ul {
+  margin: 0;
+  padding: 0;
+}
+
+.button {
+  background: #{linkcolor};
+  border-radius: 1.125em;
+  color: #fff;
+  display: inline-block;
+  line-height: 1;
+  padding: 8px 1.5em 9px;
```

Comment:
I think it's how the font is getting rendered, the line-height is just not centered. it's just an `<a>` so probably not a button thing.